hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/action/clean/CleanPlanActionExecutor.java

```diff
@@ -58,8 +59,34 @@ public CleanPlanActionExecutor(HoodieEngineContext context,
     this.extraMetadata = extraMetadata;
   }
 
-  protected Option<HoodieCleanerPlan> createCleanerPlan() {
-    return execute();
+  private int getCommitInfo() {
+    Option<HoodieInstant> lastCleanInstant = table.getActiveTimeline().getCleanerTimeline().filterCompletedInstants().lastInstant();
+    HoodieTimeline commitTimeline = table.getActiveTimeline().getCommitTimeline().filterCompletedInstants();
```

Comment:
do you want `getCommitsTimeline()`? instead? please check again what actions are actually filtered across COW and MOR

Reply:
`getCommitsTimeline()` filters delta_commit as well along with commit and replace_commit. I do not think delta_commit actions are required here since cleaner will only clean base parquet files. Pls correct me if I am wrong here.

Reply:
nope. cleaner could clean log files too.